test/helpers/IntegrationHelper.js

```diff
@@ -799,6 +835,11 @@ global.integration = function (definitions) {
                 if (!options.player2) {
                     options.player2 = {};
                 }
+
+                // validate supplied parameters
+                validatePlayerOptions(options.player1, 'player1', options.phase);
+                validatePlayerOptions(options.player2, 'player2', options.phase);
```

Comment:
Awesome!! Thanks so much, been wanting this for a while

Reply:
With pleasure!